orchagent/fdborch.cpp

```diff
@@ -168,11 +168,23 @@ void FdbOrch::update(sai_fdb_event_t        type,
                    type, update.entry.mac.to_string().c_str(),
                    entry->bv_id, bridge_port_id);
 
+
     if (bridge_port_id &&
         !m_portsOrch->getPortByBridgePortId(bridge_port_id, update.port))
     {
-        SWSS_LOG_ERROR("Failed to get port by bridge port ID 0x%" PRIx64 ".",
+        if (type == SAI_FDB_EVENT_FLUSHED)
+        {
+            /* in case of flush - can be ignored due to a race */
+            /* there are notifications about FDB FLUSH (syncd/sai_redis) on port which was already removed
```

Comment:
Please combine the comments from previous line and have as a single block of comment.